gwos/contracts/gw-utils/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "gw-utils"
+version = "0.1.0"
+authors = ["Nervos Network"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ckb-std = "0.9.0"
+gw-types = { git = "https://github.com/nervosnetwork/godwoken.git", rev = "91c324544424292b4d715ce376d31bc45aa3cb5d", default-features = false }
```

Comment:
Do we plan to migrate to using path dependencies to reference godwoken crates?

Reply:
The contracts are built within a docker container, I am thinking of making a `no-std` specific version of `gw-types` and `gw-common` crates.